doc/user/content/release-notes/_index.md

```diff
@@ -11,29 +11,36 @@ menu:
 
 ### Broader support for service accounts
 
-v25.2.5 allows setting the service account name materialize will use in the
-MZ k8s resource. Along with this we allow setting labels and annotations
-on service accounts we create. This should allow us to more tightly integrate with
-authorization best practices in Azure.
+Starting in v25.2.5, Materialize supports:
 
-Along with this the AWS specific `environmentd_iam_role_arn` field on the Materialize CRD will be
-depreciated. Please set the `eks.amazonaws.com/role-arn` service account annotation instead.
```

Comment:
guessing deprecated instead of depreciated?  and took a guess that it's currently deprecated started in 25.2.5 instead of will be in the future?